platform_versioned_docs/version-23.1.0/enterprise/advanced-topics/use-iam-role.mdx

```diff
@@ -13,22 +13,131 @@ AWS-based customers can configure Tower to interact with other AWS Services like
 
 ## Configure the Tower IAM policy
 
-!!! tip "Assumptions in Provided Policies"
-
-    The provided policies were designed with certain assumptions:
+:::note
+The provided policies were designed with certain assumptions:
 
-    1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
-    2. **Trust Policy**: The Role should be assumable by EC2, ECS, EKS, and only specifically-named IAM actors.
+1. **IAM Policy**: Tower and Nextflow should have whole access to identified S3 Buckets.
```

Comment:
This is the Tower IAM role. I don't think Nextflow assumptions matter, nor do we need to mention them.